.eslintignore

```diff
@@ -1,10 +1,7 @@
-build/*
-coverage/*
-dist/*
-public/*
-node_modules/*
-VETS-ONLY/*
-src/js/*
-fractalEntry.js
-vets-sketchapp/*
-/src/utils/*
+**/build/*
```

Comment:
Not sure how we'd want to handle this if our linting was in a separate package- i guess we could try to allow each package pass in its own `eslintignore` but I don't know if we can merge that with another set of rules